dask/bytes/tests/test_s3.py

```diff
@@ -32,43 +35,142 @@
+                                      1: '2015-01-10 20:53:28',
+                                      2: '2015-01-10 20:43:41',
+                                      3: '2015-01-10 20:35:31',
+                                      4: '2015-01-10 20:52:58'},
+            'passenger_count': {0: 1, 1: 1, 2: 1, 3: 1, 4: 1},
+            'trip_distance': {0: 1.59, 1: 3.3, 2: 1.8, 3: 0.5, 4: 3.0},
+            'pickup_longitude': {0: -73.993896484375,
+                                 1: -74.00164794921875,
+                                 2: -73.96334075927734,
+                                 3: -74.00908660888672,
+                                 4: -73.97117614746094},
+            'pickup_latitude': {0: 40.7501106262207,
+                                1: 40.7242431640625,
+                                2: 40.80278778076172,
+                                3: 40.71381759643555,
+                                4: 40.762428283691406},
+            'RateCodeID': {0: 1, 1: 1, 2: 1, 3: 1, 4: 1},
+            'store_and_fwd_flag': {0: 'N', 1: 'N', 2: 'N', 3: 'N', 4: 'N'},
+            'dropoff_longitude': {0: -73.97478485107422,
+                                  1: -73.99441528320312,
+                                  2: -73.95182037353516,
+                                  3: -74.00432586669923,
+                                  4: -74.00418090820312},
+            'dropoff_latitude': {0: 40.75061798095703,
+                                 1: 40.75910949707031,
+                                 2: 40.82441329956055,
+                                 3: 40.71998596191406,
+                                 4: 40.742652893066406},
+            'payment_type': {0: 1, 1: 1, 2: 2, 3: 2, 4: 2},
+            'fare_amount': {0: 12.0, 1: 14.5, 2: 9.5, 3: 3.5, 4: 15.0},
+            'extra': {0: 1.0, 1: 0.5, 2: 0.5, 3: 0.5, 4: 0.5},
+            'mta_tax': {0: 0.5, 1: 0.5, 2: 0.5, 3: 0.5, 4: 0.5},
+            'tip_amount': {0: 3.25, 1: 2.0, 2: 0.0, 3: 0.0, 4: 0.0},
+            'tolls_amount': {0: 0.0, 1: 0.0, 2: 0.0, 3: 0.0, 4: 0.0},
+            'improvement_surcharge': {0: 0.3, 1: 0.3, 2: 0.3, 3: 0.3, 4: 0.3},
+            'total_amount': {0: 17.05, 1: 17.8, 2: 10.8, 3: 4.8, 4: 16.3}}
+    sample = pd.DataFrame(data)
+    df = sample.take(np.arange(5).repeat(10000))
+    file = io.BytesIO()
+    sfile = io.TextIOWrapper(file)
+    df.to_csv(sfile, index=False)
+
+    key = "nyc-taxi/2015/yellow_tripdata_2015-01.csv"
     client = boto3.client('s3')
-    client.create_bucket(Bucket=bucket, ACL='public-read-write')
-    for f, data in files.items():
-        client.put_object(Bucket=bucket, Key=f, Body=data)
-
-    yield DaskS3FileSystem(anon=True)
-
-    for f, data in files.items():
-        try:
-            client.delete_object(Bucket=bucket, Key=f, Body=data)
-        except Exception:
-            pass
-        finally:
-            m.stop()
-            httpretty = pytest.importorskip('httpretty')
```

Comment:
FYI, this importorskip was redundant since the module is skipped when httpretty isn't available: https://github.com/dask/dask/pull/4276/files#diff-4902633c800b54775a65c28a00366791R14